src/Tools/ExternalAccess/OmniSharp/Analyzers/OmnisharpAnalyzerLoaderFactory.cs

```diff
@@ -11,6 +11,7 @@ internal static class OmnisharpAnalyzerAssemblyLoaderFactory
     {
         public static IAnalyzerAssemblyLoader CreateShadowCopyAnalyzerAssemblyLoader(string? baseDirectory = null)
         {
+            // jason for review
```

Comment:
Should be fine. It looks like we never actually passed a baseDirectory in the first place. My only concern would be if the name is too long and we get closer to MAX_PATH, but we'll get feedback easily enough if that's the case.